kit/platform/errors/error_user.go

```diff
@@ -1,103 +1,115 @@
-package tenant
+package errors
 
 import (
 	"fmt"
-
-	"github.com/influxdata/influxdb/v2/kit/platform/errors"
 )
 
 const MinPasswordLen int = 8
+const MaxPasswordLen = 72
+const SpecialChars = `!@#$%^&*()_+`
```

Comment:
I'm kind of weirded out about these constants being in the `errors` package, but they were already here so I won't request a change.

Reply:
Yes, the whole import structure of this bothers me, but I don't have anything better.